fission-web-server/README.md

```diff
@@ -52,11 +52,23 @@ To mimic the full "fission stack" for local development, you can use the include
    -  `docker compose exec dns-auth pdnsutil create-zone runfission.test`
    -  `docker compose exec dns-auth pdnsutil create-zone fissionuser.test`
    -  `docker compose exec dns-auth pdnsutil create-zone fissionapp.test`
-4. Point your local DNS resolver to localhost.
+4. Create some initial DNS records:
+   - `docker compose exec dns-auth pdnsutil add-record runfission.test. @ A "127.0.0.1"`
+   - `docker compose exec dns-auth pdnsutil add-record fissionuser.test. gateway A "127.0.0.1"`
+   - `docker compose exec dns-auth pdnsutil add-record fissionapp.test. gateway A "127.0.0.1"`
+5. Point your local DNS resolver to localhost.
    - on macOS: this is under System Preferences > Network > Advanced.
```

Comment:
Note that for me I added 127.0.0.1 and also 192.168.1.1 (default DNS for me if none specified manually).  Without this if the container is down I can't reach anything, like `git pull` will hang for example.